network-primitives/scripts/create_bootstrap_node_compose_file.sh

```diff
@@ -23,6 +23,8 @@ services:
   dsn-bootstrap-node:
     image: ghcr.io/\${NODE_ORG}/bootstrap-node:\${NODE_TAG}
     restart: unless-stopped
+    environment:
+      - RUST_LOG=info
```

Comment:
This also works:
```suggestion
      RUST_LOG: info
```

Reply:
yes, was keeping it consistent with other envs defined for data_dog.